tests/acceptance/fulfillments-test.js

```diff
@@ -59,3 +58,41 @@ test("should navigate back to fulfillment list when parent route visit has multi
   const urlToMatch = `/route-plans/${routePlan.get("id")}/route-visits/${routeVisit.get("id")}`;
   assert.equal(currentURL(), urlToMatch);
 });
+
+test("displays Track Inventory button in case fulfillment of an sales order", async function(assert) {
```

Comment:
Let's rename to: displays track inventory button if fulfillment is a sales order.